templates/hazelcast5-ec2/inventory_plan.yaml

```diff
@@ -6,10 +6,15 @@ basename: <id>-<rnd:5>
 owner: <id>
 region: eu-central-1
 availability_zone: eu-central-1a
-vpc_id: vpc-002b5a4e5f8b8ece2
-internet_gateway_id: igw-02b8fe3ab75871205
-# Change the '20' to a different octet to prevent running into conflicts.
+#Engineering account VPC
+vpc_id: vpc-094e507d79a701227 
+#Engineering account igw
+internet_gateway_id: igw-03696757cca398137
+#Change the '20' to a different octet to prevent running into conflicts.
 cidr_block: 10.0.20.0/24
+#Change team information
+team: Cloud
```

Comment:
Are we meant to update this after we `perftest create` ? If so I think we should add something in the documentation somewhere to make this clear.

Reply:
➕

Reply:
thanks and yes, this needs to be updated before inventory apply, updated readme accordingly